src/basic/fs-util.c

```diff
@@ -855,6 +861,14 @@ int chase_symlinks(const char *path, const char *original_root, unsigned flags,
 
                 /* Determine length of first component in the path */
                 n = strspn(todo, "/");                  /* The slashes */
+
+                if (n > 1) {
+                        /* If this is more than one slash then skip the all but one silently, so that the
+                         * when we are done with it we have a normalized path with no multiple slashes. */
```

Comment:
I think there are a couple of extra `the` in the above comment. If you want to keep the same structure of the sentence, maybe something like the following reads better.
```
/* If there are more than one slash, skip all but one silently, so that, 
   when we are done with it, we have a normalized path with no multiple slashes. */
```

Reply:
will fix